package.json

```diff
@@ -80,8 +80,8 @@
     "formik": "2.2.9",
     "history": "5.0.0",
     "http-server": "0.13.0",
-    "jotai": "1.1.3",
-    "jotai-query-toolkit": "0.1.3",
+    "jotai": "https://pkg.csb.dev/pmndrs/jotai/commit/c6868435/jotai",
```

Comment:
will update when jotai is released with [my change (it's been merged)](https://github.com/pmndrs/jotai/pull/653)